### DISCUSSION TITLE
Welcome to conundrum-quest Discussions!

### DISCUSSION THREAD

**Discussion by wispyco** [wispyco/conundrum-quest#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leplerjacob** [comment]
Hi there! My name is Jacob. I am a relatively new software engineer. I understand some React, Ruby on Rails, and a lot of other tools and frameworks. I'm based in Washington, DC. I am extremely excited to get working on open source projects, so here I am :). I've forked this repo and started peeking around in order to tackle some issues.

**Reply by leplerjacob** [reply]
Not sure if there is anything I can help with on this...but if there is i'd be psyched to know.
